features/raadpleeg-verblijfplaats-met-periode/gba/test/foutsituaties-gba.feature

```diff
@@ -0,0 +1,418 @@
+
+    @fout-case
+    Abstract Scenario: De datumTot parameter heeft een ongeldige waarde
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde              |
+      | type                | RaadpleegMetPeriode |
+      | burgerservicenummer | 000000012           |
+      | datumVan            | 2018-01-01          |
+      | datumTot            | <datum tot>         |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                      |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1 |
+      | title    | Een of meerdere parameters zijn niet correct.               |
+      | status   | 400                                                         |
+      | detail   | De foutieve parameter(s) zijn: datumTot.                    |
+      | code     | paramsValidation                                            |
+      | instance | /haalcentraal/api/brphistorie/verblijfplaatshistorie        |
+      En heeft het object de volgende 'invalidParams' gegevens
+      | code | name     | reason                        |
+      | date | datumTot | Waarde is geen geldige datum. |
+
+      Voorbeelden:
+      | datum tot      |
+      | 20190101       |
+      | 01/02/2019     |
+      | 1 januari 2019 |
+      | 2019-1-1       |
+      | 2019-01-001    |
+      | 2019-01-01-    |
+      | -2019-01-01    |
+      | 2019-13-01     |
+      | 2019-06-31     |
+      | 2019-02-29     |
+      | 01-01-2019     |
+      | --             |
+      | - -            |
+
+  Rule: datumTot moet na datumVan liggen
+
+    @fout-case
+    Abstract Scenario: De datumTot parameter ligt niet na datumVan
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde              |
+      | type                | RaadpleegMetPeriode |
+      | burgerservicenummer | 000000012           |
+      | datumVan            | <datum van>         |
+      | datumTot            | <datum tot>         |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                      |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1 |
+      | title    | Een of meerdere parameters zijn niet correct.               |
+      | status   | 400                                                         |
+      | detail   | De foutieve parameter(s) zijn: datumTot.                    |
+      | code     | paramsValidation                                            |
+      | instance | /haalcentraal/api/brphistorie/verblijfplaatshistorie        |
+      En heeft het object de volgende 'invalidParams' gegevens
+      | code         | name     | reason                            |
+      | invalidParam | datumTot | datumTot moet na datumVan liggen. |
```

Comment:
'invalidParam' is geen code. Dit is datum gerelateerd en dient dus 'date' te zijn.

Reply:
ik heb dit gekopieerd van de foutcases feature van bewoning. 

Hier blijkt een verschil te zitten in de verwerking door de gba-api en de proxy-api.

https://github.com/BRP-API/Haal-Centraal-BRP-bewoning/blob/master/features/raadpleeg-bewoning-met-periode/gba/fout-cases-gba.feature
https://github.com/BRP-API/Haal-Centraal-BRP-bewoning/blob/master/features/raadpleeg-bewoning-met-periode/fout-cases.feature

De gba-API geeft code "date", de proxy geeft code "invalidParam".

Aangezien die een gba-feature is, maak ik er "date" van